webanno-constraints/src/main/java/de/tudarmstadt/ukp/clarin/webanno/constraints/ConstraintsServiceImpl.java

```diff
@@ -217,17 +217,15 @@ public ParsedConstraints loadConstraints(Project aProject)
                     if (merged.getImports().containsKey(e.getKey()) && !e.getValue()
                             .equalsIgnoreCase(merged.getImports().get(e.getKey()))) {
                         // If detected, notify user with proper message and abort merging
-                        StringBuilder errorMessage = new StringBuilder();
-                        errorMessage.append("Conflict detected in imports for key \"");
-                        errorMessage.append(e.getKey());
-                        errorMessage.append("\", conflicting values are \"");
-                        errorMessage.append(e.getValue());
-                        errorMessage.append("\" & \"");
-                        errorMessage.append(merged.getImports().get(e.getKey()));
-                        errorMessage.append(
-                                "\". Please contact Project Admin for correcting this. Constraints feature may not work.");
-                        errorMessage.append("\nAborting Constraint rules merge!");
-                        throw new ParseException(errorMessage.toString());
+                        String errorMessage = "Conflict detected in imports for key \"" +
```

Comment:
I am not sure the changes here improve the situation. It would probably be best to change this into a String.format(...) call.